tests/lib/ecma-features.js

```diff
@@ -43,12 +43,24 @@ var assert = require("chai").assert,
 var FIXTURES_DIR = "./tests/fixtures/ecma-features";
 // var FIXTURES_MIX_DIR = "./tests/fixtures/ecma-features-mix";
 
+var filesWithOutsandingTSIssues = [
+    "jsx/embedded-tags", // https://github.com/Microsoft/TypeScript/issues/7410
+    "jsx/namespaced-attribute-and-value-inserted", // https://github.com/Microsoft/TypeScript/issues/7411
+    "jsx/namespaced-name-and-attribute", // https://github.com/Microsoft/TypeScript/issues/7411
+    "jsx/test-content", // https://github.com/Microsoft/TypeScript/issues/7471
+    "jsx/multiple-blank-spaces"
```

Comment:
I have not had chance to report this one yet, but it is almost certainly another bug with some internal TypeScript utilities. The body of the AST is correct, but we cannot currently populate tokens array properly because of the issue.